tests/worker_deployment_test.go

```diff
@@ -1021,6 +1021,170 @@ func (s *WorkerDeploymentSuite) TestSetCurrentVersion_Unversioned_PromoteUnversi
 	s.verifyTaskQueueVersioningInfo(ctx, currentVars.TaskQueue(), worker_versioning.UnversionedVersionId, "", 0)
 }
 
+func (s *WorkerDeploymentSuite) TestSetCurrentVersion_Concurrent_DifferentVersions() {
+	s.OverrideDynamicConfig(dynamicconfig.WorkflowExecutionMaxInFlightUpdates, 100)
```

Comment:
I think this test should use the default value for this (which is 10). What do you think?